bigchaindb/core.py

```diff
@@ -722,10 +660,14 @@ def get_last_voted_block(self):
                 r.table('bigchain', read_mode=self.read_mode)
                 .get(last_block_id))
 
-        return res
+        return Block.from_dict(res)
 
     def get_unvoted_blocks(self):
-        """Return all the blocks that has not been voted by this node."""
+        """Return all the blocks that have not been voted on by this node.
+
+        Returns:
+            block (list(dict)): a list of unvoted blocks
```

Comment:
Should probably be ``:obj:`list` of `:obj:`dict``?

In any case, why does this return a list of dicts, when it could be returning a list of Blocks?